chris-mike/server.js

```diff
@@ -173,6 +200,12 @@ function loadArticles() {
 function loadDB() {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
   // Put your response here...
+  //
+  // 2 - 5
```

Comment:
Is only 3 and 4. It does not interact with the front end at all. Just setting up a database. 
-Erik